packages/components/carousel/src/__tests__/Carousel.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import Carousel from '../carousel';
+
+const defaultProps = {
+  images: [
+    {
+      src: 'data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" width="300" height="300"><rect width="100%" height="100%" fill="white" stroke="black" stroke-width="2"/></svg>',
```

Comment:
A proper way to use icons could be adding them to the icons package in Douro-ui, put that package as a dependency of this one, and release them.